site/zenodo_rdm/generators.py

```diff
@@ -18,6 +18,9 @@
 media_files_management_action = action_factory("manage-media-files")
 media_files_permission = Permission(media_files_management_action)
 
+files_management_action = action_factory("zenodo-manage-files")
```

Comment:
minor: not a big fan of the name, since it hints to a more general "file manager" role. Maybe something like `manage-external-doi-files`?

Reply:
I agree it is a bit generic now. But in this case, the action itself (and the `Generator`) are not aware of DOIs at all. That comes from the conditional generator `IfExternalDOIRecord`

I can still change it though, but maybe drop a comment and say "To be used for ExternalDOIs only"?